src/core/compile.ts

```diff
@@ -58,5 +60,5 @@ export function compile(
     }
 
     // Make HiGlass models for individual tracks
-    createHiGlassModels(JSON.parse(JSON.stringify(specCopy)), trackInfos, setHg, theme);
+    createHiGlassModels(JSON.parse(JSON.stringify(specCopy)), trackInfos, callback, theme);
```

Comment:
Probably should consider changing this API to something `async`/`await` based like most things in Gosling. Lots of HiGlass internals are based on callbacks but IMO it's confusing to switch between callbacks and async/await.

Reply:
I agree. I would use the callback as is for now, but that is something that we can definitely change.